from django_encrypt_file import EncryptionService
from exceptions import ValidationError